server/plugin_test.go

```diff
@@ -44,12 +45,12 @@ func TestPlugin(t *testing.T) {
 	personalMeetingRequest := httptest.NewRequest("POST", "/api/v1/meetings", strings.NewReader("{\"channel_id\": \"thechannelid\", \"personal\": true}"))
 	personalMeetingRequest.Header.Add("Mattermost-User-Id", "theuserid")
 
-	endedPayload := `{"event": "meeting.ended", "payload": {"object": {"id": "234"}}}`
-	validStoppedWebhookRequest := httptest.NewRequest("POST", "/webhook?secret=thewebhooksecret", strings.NewReader(endedPayload))
+	validWebhookRequest := httptest.NewRequest("POST", "/webhook?secret=thewebhooksecret", strings.NewReader("id=234&uuid=1dnv2x3XRiMdoVIwzms5lA%3D%3D&status=ENDED&host_id=iQZt4-f1ZQp2tgWwx-p1mQ"))
+	validWebhookRequest.Header.Add("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
https://github.com/mattermost/mattermost-plugin-zoom/pull/111 did migrate the code to use `application/json` instead of `application/x-www-form-urlencoded`. As far as I understand this was done to be compliant with a breaking change in the Zoom API. What is the reasoning behind:

1. Changing the code to be also working with `application/x-www-form-urlencoded`?
2. Modifying the tests to use a schema that the actual API does not use any longer?

cc @mickmister Please let me know if I misunderstand your PR.